contracts/mock/Mock0x.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import "./MockExchange.sol";
+
+contract Mock0x is MockExchange {
+    constructor(address[] memory _tokens) MockExchange(_tokens) {}
+
+    fallback() external payable {
+        (address from, address to, uint256 amount) = abi.decode(
+            msg.data,
+            (address, address, uint256)
+        );
+
+        swapTokens(from, to, amount);
+
+        assembly {
```

Comment:
can we get some comments for this?

Reply:
[added](https://github.com/lindy-labs/sc_solidity-contracts/pull/219/commits/cb42006019b4bec99a95050798be8ec0e69c3c9e)